packaging/debian-sid/changelog

```diff
@@ -1,3 +1,22 @@
+snapd (2.51.2-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1929842
+    - snapstate: remove temporary snap file for local revisions early
+    - interface: allows reading sd cards internal info from block-
+      devices interface
+    - o/ifacestate: do not visit same halt tasks in waitChainSearch to
+      avoid cycles
```

Comment:
oh well, I thought I had updated the summary of that one, but apparently didn't, it's a bit misleading

Reply:
What should it say instead? I can fix the changelog trivially

Reply:
to avoid slow convergence (or unlikely cycles)